plugin.video.npogemist/resources/lib/npo_start.py

```diff
@@ -0,0 +1,483 @@
+# -*- coding: utf-8 -*-
+"""A Kodi plugin for watching NPO, using the new npo-start API."""
+import datetime
+import re
+import time
+import calendar
+import urllib
+
+import start_api_retriever
+
+import sys
+sys.path.append("../../")
```

Comment:
remove this
the bellow import looks correct to me so there is no need to fiddle with `sys.path`